.github/workflows/docker_build_push.yml

```diff
@@ -0,0 +1,31 @@
+name: Build and push Docker Image
+
+on:
+  push:
+    paths:
+      - 'Dockerfile'
+      - 'package.json'
+      - 'package-lock.json'
+      - '.github/workflows/docker_build_push.yml'
+
```

Comment:
No estoy seguro de que tengas que poner entre comillas simples los nombres de los paths, yo en mi proyecto y en los que he visto se pone directamente el nombre del archivo.

Reply:
Le he echado un vistazo a la [documentación](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-including-paths) de las GitHub Actions y he encontrado algún ejemplo con las comillas. A lo mejor funciona sin las comillas pero lo voy a dejar así que en principio me funciona correctamente. :smile: